tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/DnDTest.java

```diff
@@ -73,9 +75,9 @@ private void dropStringToCentralBrowser(String whatToSend) {
 	}
 
 	private boolean installationStartedCheck() {
-		new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
 		try {
-			new DefaultShell("Install New Software");
+			new WaitUntil(new ShellIsActive("Progress Information"), TimePeriod.LONG);
+			new WaitUntil(new ShellIsActive("Install New Software"), TimePeriod.LONG);
```

Comment:
Instead of ShellIsActive, use ShellIsAvailable - its safer.

Reply:
Fixed